src/lib/utils.ts

```diff
@@ -27,3 +27,15 @@ export const getMessagesFromError = (errBody: CustomErrorBody): string[] => {
 
   return errBody.errors.map(err => getAllErrMessages(err)).flat();
 };
+
+/**
+ * Get a random default avatar
+ * @returns string path to an avatar .png, suitable for use as src for a Next Image component
+ */
+export const getDefaultAvatarSrc = (): string => {
```

Comment:
If you've seen the current live portal, one funny thing that happens is when you refresh the leaderboard or any other pages, the default user images are constantly re-generated since they're completely random so they switch back and forth for the same users. 

This isn't a big deal, but I was thinking that one way we could address this is by taking in an optional parameter of the user's name or uuid and hashing it to an index mod 9 for one of the profile images so that every user always shows up with the same default image and we still have a good variety with all of the different image options available. Let me know what you think abt this. Maybe we can try it out in a future issue once we have testing data on the leaderboard to see multiple users at once.